File: client/src/app/api/models/meter-dto.ts
import { MeterType } from './meter-type';

export type MeterDto = {
    id?: number;
    owner: (string) | (null);
    location: (string) | (null);
    meterNumber: (string) | (null);
    comment?: (string) | (null);
    type?: MeterType;
  };

File: server/Application/DTOs/MeterDto.cs
using Domain.Enums;

namespace Application.DTOs;

public class MeterDto
{
    public int Id { get; set; }
    public required string Owner { get; set; }
    public required string Location { get; set; }
    public required string MeterNumber { get; set; }
    public string? Comment { get; set; }
    public MeterType Type { get; set; }
}
